torch/csrc/distributed/c10d/symm_mem/nvshmem_extension.cu

```diff
@@ -945,6 +954,66 @@ void _make_a2a_exchange_plan(
       stream);
 }
 
+void _all_to_all_get(
```

Comment:
do you need a separate api for this, or just marking some inputs `optional` would allow you to reuse one of the existing all_to_alls?